bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/DefaultEntryLogger.java

```diff
@@ -968,12 +968,18 @@ public void scanEntryLog(long entryLogId, EntryLogScanner scanner) throws IOExce
         // Buffer where to read the entrySize (4 bytes) and the ledgerId (8 bytes)
         ByteBuf headerBuffer = Unpooled.buffer(4 + 8);
         BufferedReadChannel bc;
+        boolean throwExceptionWhenGetChannel = false;
         // Get the BufferedChannel for the current entry log file
         try {
             bc = getChannelForLogId(entryLogId);
         } catch (IOException e) {
             LOG.warn("Failed to get channel to scan entry log: " + entryLogId + ".log");
+            throwExceptionWhenGetChannel = true;
```

Comment:
Why not call `headerBuffer.release()` here?

Reply:
Thanks for review. I think the code function is the same. release resource in finally code block is more like the java style.

Reply:
agree with @AnonHxy